build.sbt

```diff
@@ -621,3 +621,12 @@ def customCommands: Seq[Setting[_]] = Seq(
       state
   }
 )
+
+inThisBuild(Seq(
+  whitesourceProduct                   := "Lightbend Reactive Platform",
```

Comment:
Does this mean you're offering subscriptions for this product? This is novel, it's not even the case for scalac. Please, make this clearer.

Reply:
Yeah scala, sbt, zinc are products of RP, and Lightbend provides commercial support for them.